extension/src/popup/basics/layout/Box/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { addStyleClasses } from "popup/helpers/addStyleClasses";
+
+import "./styles.scss";
+
+interface BoxProps {
+  display?: "grid" | "flex";
+  children: React.ReactElement | React.ReactElement[];
+  gridCellWidth?: string;
+  gapHorizontal?: string;
+  gapVertical?: string;
+  isFlexRow?: boolean;
+}
+
+export const Box: React.FC<BoxProps> = ({
```

Comment:
This component attempts to create a layout system that we can use to build the most common layouts. It's very limited right now, but we can keep growing it if the idea seems OK.